frontend/package.json

```diff
@@ -20,11 +20,12 @@
   },
   "dependencies": {
     "@formkit/auto-animate": "^0.8.2",
+    "@nuxt/content": "^3.5.1",
     "@nuxt/image": "1.10.0",
     "@nuxt/ui": "3.1.0",
     "@nuxtjs/mdc": "0.16.1",
-    "@nuxtjs/seo": "3.0.3",
-    "@vueuse/nuxt": "13.1.0",
+    "@nuxtjs/seo": "^3.0.3",
```

Comment:
Nope. We pin nuxt related dependencies to eliminate bugs in newer versions. Remove `^` completly